docs/pages/_document.js

```diff
@@ -143,6 +144,12 @@ export default class MyDocument extends Document {
         </Head>
         <body>
           {getInitColorSchemeScript({ enableSystem: true })}
+          {getMuiInitColorSchemeScript({
```

Comment:
Created an issue to keep track of this: https://github.com/mui/material-ui/issues/33853

Reply:
Thanks, we can wait to see if people are hit by this problem.

Reply:
> Why not share the same one? I mean, it's not clear to me why as single
> 
> ```js
> var mode = localStorage.getItem('mui-system-mode');
> ```
> 
> is not enough.

Set the storage key to muidocs so that other websites that uses the default values do not alter our docs.

Reply:
What do you mean by "other website"? The local storage is bound to a specific origin https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage

Reply:
> What do you mean by "other website"? The local storage is bound to a specific origin https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage.
> 
> I guess to rephrase my question. I'm trying to understand why we don't store the prefered color mode once per origin.

Yeah, you are right. There is no need to use a custom key, I will remove it. But note that we need both of the scripts like this because they target different storage key:

```js
{getInitColorSchemeScript({ enableSystem: true })} // for Joy
{getMuiInitColorSchemeScript({ enableSystem: true })} // for Material UI
```